templates/monitoring/blackbox/target.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: applicationmonitoring.integreatly.org/v1alpha1
+kind: BlackboxTarget
+metadata:
+  name: {{ index .ExtraParams "name"  }}
+  namespace: {{ .Namespace }}
+  labels:
+    monitoring-key: {{ .MonitoringKey }}
```

Comment:
Can we switch to:

- `index .Params "name"`
- `index .Params "Namespace"`
- `index .Params "MonitoringKey"`

Save me rebasing and changing in #195

Reply:
Or nm, might be less work if I just tackle the change in #195